src/main/scala/org/hammerlab/guacamole/commands/SomaticStandardCaller.scala

```diff
@@ -207,7 +212,9 @@ object SomaticStandard {
         Likelihood.likelihoodsOfAllPossibleGenotypesFromPileup(
           filteredNormalPileup,
           Likelihood.probabilityCorrectIgnoringAlignment,
-          normalize = true).toMap
```

Comment:
passing an arg, closing a function call, and doing a full collection transformation, all on one line / as an afterthought: too much, imho.

Something like `.toMap` is many lines of code in some languages; scala's concision for the option of writing very little, and having amazing one-liners, but obviously "can" != "should" in all cases.